### PR TITLE
Tech/refactor

### PR DESCRIPTION
**Pull request by sidlynx**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-users`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

Move tests from __tests__ folder so that every file tested will have it's test file in the same root with the same name suffixed with 'spec'

### CODE REVIEW

**Review by kqesar — CHANGES_REQUESTED** [review]
Fix conflicts and keep only commit which concerns your work

**Review by kqesar — APPROVED** [review]

File: packages/manager/apps/pci-users/src/__tests__/__test-utils__/contextRenders.tsx [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-      <QueryClientProvider client={queryClient}>
-        <ApplicationProvider environment={environment} shell={shell}>
-          <ContainerProvider>{children}</ContainerProvider>
-        </ApplicationProvider>
-      </QueryClientProvider>
-    </I18nextProvider>,
-    renderOptions,
-  );
-};
-
```
Comment (seven-amid) [inline packages/manager/apps/pci-users/src/__tests__/__test-utils__/contextRenders.tsx:33]: what's the reason behind this file deletion ?

Comment (sidlynx) [reply packages/manager/apps/pci-users/src/__tests__/__test-utils__/contextRenders.tsx:33]: It's not used, i was running vitest in watch mode while refactoring tests, no fail was detected, and i believe that Yoann made a copy/paste of an existing project where those utilities were used

Comment (seven-amid) [reply packages/manager/apps/pci-users/src/__tests__/__test-utils__/contextRenders.tsx:33]: Alright

**Pull request merged**